hmmlearn/base.py

```diff
@@ -102,6 +102,9 @@ def __init__(self, n_components=1, startprob=None, transmat=None,
         self.transmat_prior = transmat_prior
         self.algorithm = algorithm
         self.random_state = random_state
+        self.logprob_ = None
+        self.n_iterations_performed = None
+        self.logprob_delta = None
```

Comment:
Looks good! 

I think we should also amend `BaseHMM` docstring to reflect the newly added attributes (similar to how it's done for [`sklearn.mixture.GMM`](http://scikit-learn.org/stable/modules/generated/sklearn.mixture.GMM.html#sklearn.mixture.GMM)). Sorry for not mentioning this right away.